node/src/chain_spec.rs

```diff
@@ -59,29 +67,67 @@ where
 /// Generate the session keys from individual elements.
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn pop_session_keys(keys: AuraId) -> pop_runtime::SessionKeys {
-	pop_runtime::SessionKeys { aura: keys }
+pub fn pop_devnet_session_keys(keys: AuraId) -> pop_runtime_devnet::SessionKeys {
+	pop_runtime_devnet::SessionKeys { aura: keys }
+}
+/// Generate the session keys from individual elements.
+///
+/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+pub fn pop_testnet_session_keys(keys: AuraId) -> pop_runtime_testnet::SessionKeys {
+	pop_runtime_testnet::SessionKeys { aura: keys }
 }
 
-pub fn development_config() -> ChainSpec {
-	// Give your base currency a unit name and decimal places
-	let mut properties = sc_chain_spec::Properties::new();
-	properties.insert("tokenSymbol".into(), "UNIT".into());
-	properties.insert("tokenDecimals".into(), 12.into());
+fn configure_for_relay(
+	relay: Relay,
+	properties: &mut sc_chain_spec::Properties,
+) -> (Extensions, u32) {
+	properties.insert("tokenSymbol".into(), "POP".into());
```

Comment:
I think this `properties.insert("tokenSymbol".into(), "POP".into());` should be based on what relay it is. `POP` token does not exist.

Reply:
Ah you are so right. This needs to be changes all across all the properties. Thanks.